doc/classes/Array.xml

```diff
@@ -144,21 +144,21 @@
 			</argument>
 			<description>
 				Returns a copy of the array.
-				If [code]deep[/code] is [code]true[/code], a deep copy is be performed: all nested arrays and dictionaries are duplicated and will not be shared with the original array. If [code]false[/code], a shallow copy is made and references to the original nested arrays and dictionaries are kept, so that modifying a sub-array or dictionary in the copy will also impact those referenced in the source array.
+				If [code]deep[/code] is [code]true[/code], a deep copy is performed: all nested arrays and dictionaries are duplicated and will not be shared with the original array. If [code]false[/code], a shallow copy is made and references to the original nested arrays and dictionaries are kept, so that modifying a sub-array or dictionary in the copy will also impact those referenced in the source array.
 			</description>
 		</method>
 		<method name="empty">
 			<return type="bool">
 			</return>
 			<description>
-				Return true if the array is empty (size==0).
+				Returns [code]true[/code] if the array is empty (size==0).
```

Comment:
Should "(size==0)" also have "[code]"?

Reply:
I had thought about just removing that parenthetical. "empty" is pretty self-explanatory.